### PR TITLE
Filipino Translation - 1 037 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Translated advanced-merging.asc with total 1,037 translated words.
Note : Code text and computer code text are not included as translated words

### CODE REVIEW

File: book/07-git-tools/sections/advanced-merging.asc [hunk 1]
```diff
@@ -462,13 +462,13 @@ index 0399cd5,59727f0..0000000
...
+Ang unang column ay nagpapakita sa iyo kung ang linya ay iba (nadagdag o inalis) sa pagitan ng ``ours'' na branch at ang file sa iyong tinatrabahong direktoryo at ang pangalawang column ay gagawa pareho sa pagitan ng ``theirs'' na branch at ang iyong tinatrabahong direktoryo na kopya.
 
-So in that example you can see that the `<<<<<<<` and `>>>>>>>` lines are in the working copy but were not in either side of the merge.
-This makes sense because the merge tool stuck them in there for our context, but we're expected to remove them.
+Kaya sa halimbawang iyon ay makikita mo na ang `<<<<<<<` and `>>>>>>>` na mga linya ay nasa tinatrabahong kopya ngunit ay hindi alinman na panig sa merge.
+Ito ay may katuturan dahil sa ang merge na kasangkapan ay natigil nila na para doon sa susunod na conteksto natin, ngunit inaasahan namin na maalis sila.
 
-If we resolve the conflict and run `git diff` again, we'll see the same thing, but it's a little more useful.
+Kung lutasin natin ang salungat at ipatakbo ang `git diff` muli, makikita natin ang parehong bagay, ngunit ito ay isang maliit na mas magagamit.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `isang maliit na mas magagamit` to `medyo mas magagamit`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 2]
```diff
@@ -490,11 +490,11 @@ index 0399cd5,59727f0..0000000
...
 ----
 
-This shows us that ``hola world'' was in our side but not in the working copy, that ``hello mundo'' was in their side but not in the working copy and finally that ``hola mundo'' was not in either side but is now in the working copy.
-This can be useful to review before committing the resolution.
+Ito ay nagpapakita na ang ``hola world'' ay nasa iyong panig ngunit wala sa tinatrabahong kopya, na ang ``hello mundo'' ay nasa kanilang panig ngunit wala sa tinatrabahong kopya at sa wakas na ``hola mundo'' ay wala sa magkabilang panig ngunit ngayon ay nasa tinatrabahong kopya.
+Ito ay maaaring magagamit sa pagsusuri bago ang pag-commit sa resolusyon.
 
-You can also get this from the `git log` for any merge to see how something was resolved after the fact.
-Git will output this format if you run `git show` on a merge commit, or if you add a `--cc` option to a `git log -p` (which by default only shows patches for non-merge commits).
+Maaari kang makakuha din nito mula sa `git log` para sa anumang merge upang makita kung paano nalutas pagkatapos sa katutuhanan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `katutuhanan` to `katotohanan`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 3]
```diff
@@ -490,11 +490,11 @@ index 0399cd5,59727f0..0000000
...
+Ang Git ay mag-output ng ganitong format kung magpatakbo ka ng `git show` sa isang merge commit, o kung ikaw ay magdagdag ng `--cc` na opsyon sa isang `git log -p` (na sa pamamagitan ng default lamang ay nagpapakita ng mga patch para sa non-merge na mga commit).
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `mag-output` to `mag-a-output`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 4]
```diff
@@ -526,76 +526,76 @@ index 0399cd5,59727f0..e1d0799
 ----
 
 [[_undoing_merges]]
-==== Undoing Merges
+==== Pag-undo sa mga merge
 
-Now that you know how to create a merge commit, you'll probably make some by mistake.
-One of the great things about working with Git is that it's okay to make mistakes, because it's possible (and in many cases easy) to fix them.
+Ngayon na alam mo na kung papaano maglikha ng isang merge commit, malamang ikaw ay gagawa ng pagkakamali.
+Isa sa mga dakilang mga bagay tungkol sa tinatrabaho gamit ang Git ay ok lang na makagawa ng mga pagkakamali, dahil ito ay posible (at sa maraming mga kaso na madaliy) sa pag-ayos nila.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `madaliy` to `madali`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 5]
```diff
@@ -526,76 +526,76 @@ index 0399cd5,59727f0..e1d0799
 ----
 
 [[_undoing_merges]]
-==== Undoing Merges
+==== Pag-undo sa mga merge
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: change capitalization `Pag-undo sa mga merge` to `Pag-undo sa mga Merge`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 6]
```diff
@@ -526,76 +526,76 @@ index 0399cd5,59727f0..e1d0799
...
-. Make the working directory look like the index.
+. Igalaw ang branch HEAD na mga puntos sa.
+  sa kasong ito, gusto naming ilipat ang `master` sa kung saan ito ay nating na merge commit (`C6`).
+. Gawin ang index na kamukha ng HEAD.
+. Gawin ang tinatrabahong direktoryo na kamukha ng index.
 
-The downside of this approach is that it's rewriting history, which can be problematic with a shared repository.
-Check out <<_git_branching#_rebase_peril>> for more on what can happen; the short version is that if other people have the commits you're rewriting, you should probably avoid `reset`.
-This approach also won't work if any other commits have been created since the merge; moving the refs would effectively lose those changes.
+Ang downside sa diskarteng ito ay ang kanyang sinusulatang muli na kasaysayan, na maaaring maging problema na may isang ibinabahaging direktoryo.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: change `downside` to `kakulangan`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 7]
```diff
@@ -526,76 +526,76 @@ index 0399cd5,59727f0..e1d0799
...
+Tingnan ang <<_git_branching#_rebase_peril>> para sa marami na mga pwedeng mangyari; ang maikling bersyon ay kung ang ibang tao ay mayroong mga commit na iyong sinusulatang muli, dapat mong marahil iwasan ang `reset`.
+Ang pamamaraan na ito ay hindi rin gagana kung ang ibang mga anumang commit ay nalikha mula nang na-merge; paglilipat sa mga ref ay epektibong mawawala ang mga pagbabago na iyon.
 
 [[_reverse_commit]]
-===== Reverse the commit
+===== Baliktad na commit
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `Baliktad na commit` to `Ibaliktad ang commit`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 8]
```diff
@@ -604,31 +604,31 @@ $ git revert ^M
...
 
-.History after re-merging a reverted merge
-image::images/undomerge-revert3.png[History after re-merging a reverted merge.]
+.Kasaysayan pagkatapos ng pagre-re-merge sa isang na revert na merge
+image::images/undomerge-revert3.png[Kasaysayan pagkatapos ng pagre-re-merge sa isang na revert na merge.]
 
-In this example, `M` and `^M` cancel out.
-`^^M` effectively merges in the changes from `C3` and `C4`, and `C8` merges in the changes from `C7`, so now `topic` is fully merged.
+Sa halimbawa na ito, `M` at `^M` ay kinansela.
+`^^M` ay epektibong nag-merge sa mga pagbabago mula sa `C3` at `C4`, at `C8` na mga merge sa pagbabago mula `C7`, kaya ngayon ang `topic` ay ganap na na-merge.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `` Ang `^^M` ``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 9]
```diff
@@ -639,7 +639,7 @@ Resolved 'hello.rb' using previous resolution.
 Automatic merge failed; fix conflicts and then commit the result.
 ----
 
-However if we run it with `-Xours` or `-Xtheirs` it does not.
+Gayunpaman kung tayo ay magpatakbo ng `-Xours` o `-Xtheirs` it ay hindi.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `` Gayunpaman kung tayo ay magpatakbo nito ng `-Xours` o `-Xtheirs` ito ay hindi. ``

**Review by josteinfrancois — COMMENTED** [review]
Hello @robin-ho . Just finished reviewing your work.

File: book/07-git-tools/sections/advanced-merging.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: change done

File: book/07-git-tools/sections/advanced-merging.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

**Comment by mind-blast** [comment]
done change all suggested words.. thank you.

**Pull request merged**